applications: nrf5340_audio: Added capabilities doc for the controller (#12060)
nrfconnect/sdk-nrf: doc/nrf/libraries/bin/bt_ll_acs_nrf53/index.rst

When the communication between the application and the network core on a device is handled by the HCI RPMsg driver, this link layer controller can be used to handle time critical low level communication and the radio.

.. note::

>    The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID #181316.

Reviewer: ```suggestion
   The LE Audio controller for nRF5340 associated with this version of the |NCS| comes with the QDID ``#181316``.
```

Author: Done

Configuration
*************

The controller can be enabled with the :Kconfig:option:`CONFIG_BT_LL_ACS_NRF53` set to ``y``.

> The number of Connected Isochronous Streams (CIS) you can use in a Connected Isochronous Group (CIG) and the number of Broadcast Isochronous Streams (BIS) you can use in a Broadcast Isochronous Group (BIG)
> are dependent on which configurations you use.

Reviewer: ```suggestion
The number of Connected Isochronous Streams (CIS) you can use in a Connected Isochronous Group (CIG) and the number of Broadcast Isochronous Streams (BIS) you can use in a Broadcast Isochronous Group (BIG) are dependent on which configurations you use.
```

Author: Done

> These are configurations like sample frequency, bitrate and number of retransmissions.

Reviewer 2: ```suggestion
The relevant configurations are bitrate and number of retransmissions.
```
Sample frequency does not affect this, the packet on air can be 120 bytes regardless if it is sampled in 48kHz or 8kHz

> For the matrix of supported CIS in a CIG and BIS in a BIG, see the `Capabilities for the LE Audio Controller Subsystem for nRF5340`_ document, which is based on the Basic Audio Profile (BAP) tables.

Reviewer: ```suggestion
See the `Capabilities for the LE Audio Controller Subsystem for nRF5340`_ document for a matrix table listing the supported CIS in a CIG and the supported BIS in a BIG.
The document is based on the Basic Audio Profile (BAP) tables.
```

Author: Done, with some modifications
